app/helpers/user_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module UserHelper
   def tag_name_options(user)
-    user.predictions.includes(:tags).pluck(:name)
+    user.predictions.includes(:tags).pluck(:name).uniq
```

Comment:
@jaysonvirissimo This (uniq) work can be done in the database. I've reworked this method to be a bit more efficient.

Reply:
Ah, thanks. That's much better. 👏🏼